salvo/tools/format_python_tools.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# This script runs the style and formatting checks
+
+set -e
+
+VENV_DIR="pyformat"
+SCRIPTPATH=$(realpath "$(dirname $0)")
+. $SCRIPTPATH/shell_utils.sh
+cd "$SCRIPTPATH"
+
+source_venv "$VENV_DIR"
+echo "Installing requirements..."
+pip install -r requirements.txt
+
+echo "Running Python format check..."
+python format_python_tools.py $1
+
+echo "Running Python3 flake8 check..."
+cd ..
+EXCLUDE="--exclude=benchmarks/tmp/*,.cache/*,*/venv/*,tools/format_python_tools.py,tools/gen_compilation_database.py,bazel-*"
+
+
+# Because of conflict with the automatic fix format script, we ignore: 
+# E111 Indentation is not a multiple of four
+# E114 Indentation is not a multiple of four (comment)
+# E501 Line too long (82 > 79 characters)
+# E124 Closing bracket does not match visual indentation
+# E125 Continuation line with same indent as next logical line
+# E126 Continuation line over-indented for hanging indent
+
+# We ignore false positives because of what look like pytest peculiarities 
+# F401 Module imported but unused
```

Comment:
We didn't get to reviewing Python code yet, but these two feel like they may be result of how we are importing code / setting up the build system. It may be valuable for us to try and address them before we silence them which can result in accumulation of technical debt. Can you help me out by pointing to the locations where these warnings come up? I might be able to suggest a simple fix.

Reply:
I copied these scripts from [nighthawk](https://github.com/envoyproxy/nighthawk/tree/master/tools) and they are largely unmodified.

Can we circle back to this because there are a few games we have to play with adding directories to the path because of the source layout and 'bazel'?    We can certainly tighten up the style and formatting checks.